util/docker/maketest.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Runs tests and reports on them to github
+
+DIR=$(cd $(dirname $0) ; pwd -P)
+source ${DIR}/docker_common.sh
+
+# Get sha sum
+USER="jgkamat"
+SUCCESS=true
+LINK_PREFIX="https://circle-artifacts.com/gh/RoboJackets/robocup-software/$CIRCLE_BUILD_NUM/artifacts/0$CIRCLE_ARTIFACTS/"
+ARTIFACT_DIR="/tmp/build_artifacts"
+
+start_pending() {
+    curl -u $USER:$TOKEN -X POST https://api.github.com/repos/robojackets/robocup-software/statuses/${SHA_SUM} -H "Content-Type: application/json" -d '{"state":"pending", "description": "A check for compiling", "context": "circle/compile", "target_url": '"\"${LINK_PREFIX}makeoutput.txt\"}"
+    curl -u $USER:$TOKEN -X POST https://api.github.com/repos/robojackets/robocup-software/statuses/${SHA_SUM} -H "Content-Type: application/json" -d '{"state":"pending", "description": "A check for cpp tests", "context": "circle/test-cpp", "target_url": '"\"${LINK_PREFIX}testcppoutput.txt\"}"
```

Comment:
`test-cpp` runs both `test-soccer` and `test-firmware`.  The way you have it covers both, but if you change it to run them as two separate items, we can get a status for each.